swagger.yaml

```diff
@@ -2764,61 +2764,6 @@ paths:
           schema:
             $ref: '#/definitions/error'
 
-  /v1/clusters/{cluster_id}/events:
```

Comment:
No sure if we can push at the moment, need to make sure the clients are not using as well. 
We are not removing v1 right now.

Reply:
placed a `hold` label.